File: nelio-alves-algoritmo-e-logica-de-programacao/python/exercicios/soma_vetor.py
soma: float; media: float
N: int

N = int(input("Quantos numeros voce vai digitar? "))

vet: [float] = [0 for x in range(N)]

for i in range(N):
	vet[i] = float(input("Digite um numero: "))

print("\nVALORES = ", end="")
for i in range(N):
	print(f"{vet[i]:.1f}  ", end="")

soma = 0
for i in range(N):
	soma = soma + vet[i]

print(f"\nSOMA = {soma:.2f}")

media = soma / N
print(f"MEDIA = {media:.2f}")

File: nelio-alves/algoritmo-e-logica-de-programacao/cpp/exercicios/soma_vetor.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main() {
    int N;
    double soma, media;

    cout << "Quantos numeros voce vai digitar? ";
    cin >> N;

    double vet[N];

    for (int i = 0; i < N; i++) {
        cout << "Digite um numero: ";
        cin >> vet[i];
    }

    cout << fixed << setprecision(1);

    cout << endl << "VALORES = ";
    for (int i = 0; i < N; i++) {
        if (i != 0) {
            cout << " ";
        }

        cout << vet[i];
    }

    cout << fixed << setprecision(2);

    soma = 0;
    for (int i = 0; i < N; i++) {
        soma = soma + vet[i];
    }

    cout << endl << "SOMA = " << soma << endl;

    media = soma / N;
    cout << "MEDIA = " << media << endl;

    return 0;
}
